common/app/routes/challenges/utils.js

```diff
@@ -90,7 +90,8 @@ function logReplacer(value) {
     const replaced = value.map(logReplacer);
     return '[' + replaced.join(', ') + ']';
   }
-  if (typeof value === 'string' && !value.startsWith('//')) {
+  if (typeof value === 'string' &&
+      !(value.charAt(0) === '/' && value.charAt(1) === '/')) {
```

Comment:
I think a Regular Expression checking for `//` at the beginning will be a more elegant solution. It also keeps the number of checks required to a minimum